packages/typespec-ts/test/modularIntegration/generated/azure/core/lro/rpc/generated/src/models/models.ts

```diff
@@ -10,6 +11,14 @@ export interface GenerationOptions {
   prompt: string;
 }
 
+export function generationOptionsSerializer(
+  item: GenerationOptions,
+): GenerationOptionsRest {
+  return {
+    prompt: item["prompt"],
```

Comment:
We are generating now a serializer for each named model. One of the benefits is that if the model is used in different operations we just need to reference the serializer instead of having to generate inline again. This will translate in bundle size savings